libs/ledger-live-common/src/featureFlags/defaultFeatures.ts

```diff
@@ -44,6 +83,45 @@ export const defaultFeatures: DefaultFeatures = {
   currencyFlare: {
     enabled: false,
   },
+  currencyAxelarMobile: {
```

Comment:
We don't need mobile-specific versions anymore you can remove them, Firebase is able to make the distinction.

Reply:
Done